pkgs/os-specific/linux/busybox/default.nix

```
@@ -32,12 +32,16 @@ let
     CONFIG_FEATURE_WTMP n
   '';
 
-  debianName = "busybox_1.30.1-5";
-  debianTarball = fetchzip {
-    url = "http://deb.debian.org/debian/pool/main/b/busybox/${debianName}.debian.tar.xz";
-    sha256 = "03m4rvs2pd0hj0mdkdm3r4m1gh0bgwr0cvnqds297xnkfi5s01nx";
+  # The debian version lacks behind the upstream version and also contains
+  # a debian-specific suffix. We only fetch the debian repository to get the
+  # default.script
+  debianVersion = "1.30.1-6";
+  debianSource = fetchgit {
+    url = "https://salsa.debian.org/installer-team/busybox.git";
+    rev = "debian/1%${debianVersion}";
```

Using fetchFromGitLab is preferred. (I did not touch the hash, it probably needs to be updated too).

```suggestion
  debianSource = fetchFromGitLab {
    domain = "salsa.debian.org";
    owner = "installer-team";
    repo = pname;
    rev = "debian/1%${debianVersion}";
```

---

`fetchFromGitlab` does not currently work when the revision contains a slash, as the line `renamed="$TMPDIR/${baseNameOf url}"` does not work as expected.